src/remote-config/remote-config-api.ts

```diff
@@ -213,6 +218,123 @@ export interface PercentCondition {
   microPercentRange?: MicroPercentRange;
 }
 
+/**
+ * Defines supported operators for custom signal conditions.
+ */
+export enum CustomSignalOperator {
+
+  /**
+   * A catchall error case.
+   */
+  UNKNOWN = 'UNKNOWN',
+
+  /**
+   * The actual value is less than the target value.
+   */
+  NUMERIC_LESS_THAN = 'NUMERIC_LESS_THAN',
+
+  /**
+   * The actual value is less than or equal to the target value.
+   */
+  NUMERIC_LESS_EQUAL ='NUMERIC_LESS_EQUAL',
+
+  /**
+   * The actual value is equal to the target value.
+   */
+  NUMERIC_EQUAL = 'NUMERIC_EQUAL',
+
+  /**
+   * The actual value is not equal to the target value.
+   */
+  NUMERIC_NOT_EQUAL = 'NUMERIC_NOT_EQUAL',
+
+  /**
+   * The actual value is greater than the target value.
+   */
+  NUMERIC_GREATER_THAN = 'NUMERIC_GREATER_THAN',
+
+  /**
+   * The actual value is greater than or equal to the target value.
+   */
+  NUMERIC_GREATER_EQUAL = 'NUMERIC_GREATER_EQUAL',
+
+  /**
+   * AT LEAST ONE of the target values is a substring of the actual custom
+   * signal value. Eg: "abc" contains the string "a", "bc".
+   */
```

Comment:
It's the *target values* (ie the things developers set up in the console when creating this condition) that can be multiple - not the *actual value* that's passed in for evaluation in the SDK.

so if you call `evaluate({myCustomSignal: 'foo'})` it would match for a condition like  `app.customSignal.myCustomSignal.contains(['bar', 'foo'])` 

As such, I think I'd prefer to keep the existing wording, though I can remove the awkward capitalizations if that helps

Reply:
A-ha, thanks, that makes sense, and sorry for the mix-up! One tiny nit, can you update `Eg:` to `E.g.,`? You can even put it in parens, like:

* Matches if at least one of the target values is a substring of the actual custom
* signal value (e.g., "abc" contains the strings "a" and "bc").